.github/.OwlBot.lock.yaml

```diff
@@ -0,0 +1,4 @@
+docker:
```

Comment:
It looks like you still have these files checked in. An easy way to fix is comment out line in gitignore, delete them temp, uncomment gitignore, readded them for yourself locally.

Reply:
I can't tell -- does it look like I fixed it?